cs3/storage/provider/v1beta1/resources.proto

```diff
@@ -141,7 +144,7 @@ enum LockType {
 //   "type" : "<LOCK_TYPE>",
 //   "h" : "<holder>",
 //   "md" : "<metadata>",
-//   "mtime" : "<Unix timestamp>"
+//   "exp" : "<Unix timestamp>"
 // }
```

Comment:
So shall we add this to the "specs"?
```suggestion
// }
// The <holder> field SHOULD have a format as follows:
// - `uid:<user>` for user-held locks
// - `app:<appname>` for apps-held (WOPI) locks
```

Reply:
OK, makes sense, as long as the generated JSON is not too opaque (which I guess is not), to not hinder debugging/operations/etc outside of Reva. Then just go ahead and drop the detailed JSON ;-) we can just say that we want to store a JSON representation without further specifications.